pkg/config/config.go

```diff
@@ -108,9 +109,12 @@ func (c *Config) ReadFile() errors.Error {
 }
 
 // ReadBytes - retrieve config from a byte
-func (c *Config) ReadBytes(data []byte) errors.Error {
-	if err := toml.Unmarshal(data, c); err != nil {
-		return errors.Wrap(errUnmarshalConfigContent, err)
+func (c *Config) ReadBytes(data []byte) (err errors.Error) {
```

Comment:
Specify the type that you return not the variable. In this you are overwriting with the returned value but it's dangerous.